execute/rowreader.go

```diff
@@ -0,0 +1,227 @@
+package execute
+
+import (
+	"database/sql"
+	"fmt"
+	"github.com/influxdata/flux"
+	"github.com/influxdata/flux/values"
+	"strconv"
+	"time"
+)
+
+const layout = "2006-01-02 15:04:05.999999999"
+
+type RowReader interface {
+	Next() bool
+	GetNextRow() ([]values.Value, error)
+	InitColumnNames([]string)
+	InitColumnTypes([]*sql.ColumnType)
+	ColumnNames() []string
+	ColumnTypes() []flux.ColType
+	SetColumns([]interface{})
+}
+
+type RowReaderImpl struct {
+	Cursor      *sql.Rows
+	columns     []interface{}
+	columnTypes []flux.ColType
+	columnNames []string
+}
+
+func (m *RowReaderImpl) SetColumns(i []interface{}) {
+	m.columns = i
+}
+
+type MySQLRowReader struct {
```

Comment:
i apologize for all the bouncing around and code re-org i'm having you do.  let's put the MySQLRowReader  and PostgresRowReader   back in stdlib/sql  and import the rest of this code from there.   All the code looks good, just need to organize it.